homeassistant/components/modbus/validators.py

```diff
@@ -127,25 +126,27 @@ def scan_interval_validator(config: dict) -> dict:
 
             for entry in hub[conf_key]:
                 scan_interval = entry.get(CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL)
-                if scan_interval < MINIMUM_SCAN_INTERVAL:
-                    if scan_interval == 0:
-                        continue
+                if scan_interval == 0:
+                    continue
+                if scan_interval < 5:  # there are not constant in core for this.
```

Comment:
```suggestion
                if scan_interval < 5:
```

Reply:
Heh I had that because I hate numbers without constants, but you are quite right.